Source/OpenTK/Exceptions.cs

```diff
@@ -33,7 +33,11 @@ namespace OpenTK
     /// <summary>
     /// This exception is thrown when a GraphicsContext property cannot be changed after creation.
     /// </summary>
+#if _NET_CORECLR
+    public class ContextExistsException : Exception
```

Comment:
Right, it was done before I felt uneasy about my changes to replace ApplicationException into just a mundane Exception.